remote_search.py

```diff
@@ -81,12 +82,13 @@ def __post_init__(self):
             AUDIO_FIELD_NAME: RemoteMediaInfo(AUDIO_FIELD_NAME, self.sound_url, MediaType.sound),
         }
         self._mapping = {
-            "SentKanji": self.sent_kanji,
-            "SentFurigana": self.sent_furigana,
-            "SentEng": self.sent_eng,
+            "Expression": self.sent_kanji,
+            "Reading": self.sent_furigana,
+            "English": self.sent_eng,
             AUDIO_FIELD_NAME: self.audio.as_anki_ref(),
             IMAGE_FIELD_NAME: self.image.as_anki_ref(),
             "Notes": self.notes,
+            "tags": self.tags,
```

Comment:
Tags can't be included in the mapping because in Anki tags are separate from fields.